Packs/FeedLOLBAS/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### New: Compare Process Execution Arguments To LOLBAS Patterns
+
+- New: This playbook takes a process name and determine its presence in the LOLBAS repository .Upon confirming the process's existence in the LOLBAS feed, the playbook proceeds to compare the incident command line against known patterns of malicious commands listed in the LOLBAS repository.
+The playbook outputs results where the similarity between the analyzed command line and the malicious patterns greater or equals the preconfigured StringSimilarity threshold. The playbook offers the flexibility to adjust this threshold through the use of the dedicated playbook input, "StringSimilarityThreshold." (Available from Cortex XSOAR 6.8.0).
```

Comment:
In the release notes  available from Cortex XSOAR 6.8.0. but in the playbook yml - its from 6.9